content/guides/overview/why-cypress.md

```diff
@@ -147,6 +148,13 @@ the developer tools you know and love.
 
 <DocsVideo src="/img/snippets/debugging.mp4"></DocsVideo>
 
+### <Icon name="chart-bar"></Icon> Analyzing test results
```

Comment:
Originally I used a pie chart icon here because the icons for the other sections are mostly kind of rounded, but then I realized we don't actually have any pie charts in cloud (AFAIK) so I changed it to a bar chart.